doc/changelog.md

```diff
@@ -79,7 +79,7 @@ Syntax: `- short text describing the change _(Your Name)_`
 - CI: unstash schemas in release pipeline in build-backend _(4ydan)_
 - _()_
 - _()_
-- Added missing error toast messages _(Daniel Steinkogler)_
+- Add decision for polygon libraries _(Daniel Steinkogler)_
 - _()_
```

Comment:
I think you deleted the existing changelog entry by mistake.

Reply:
Was not a really important changelog entry.